sdk/storage/storage-blob/src/BlobDownloadResponse.ts

```diff
@@ -443,6 +444,33 @@ export class BlobDownloadResponse implements BlobDownloadResponseParsed {
     return this.originalResponse.isSealed;
   }
 
+  /**
+   * UTC date/time value generated by the service that indicates the time at which the blob immutability policy will expire.
+   *
+   * @readonly
+   */
+  public get immutabilityPolicyExpiresOn(): Date | undefined {
```

Comment:
remember to change `BlobQueryResponse` as well

Reply:
BlobQueryResponse  doesn't expose immutabilityPolicy... properties in Swagger.

Reply:
``` ts
class BlobQueryResponse implements BlobDownloadResponseModel
```

It's a wrapper class similar to `BlobDownloadResponse` for due to download retry support.
You can discuss with the Arch board on whether we want to expose this property for the Query response..

Reply:
Query doesn't reply these two properties. I checked this through testing with fiddler.